packages/manager/modules/bm-server-components/src/consumption-tile/template.html

```diff
@@ -0,0 +1,88 @@
+                      }"
+                data-ng-class="{ 'oui-badge_error': $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.overQuota }"
+            >
+            </span>
+            <span
+                class="oui-badge"
+                data-ng-if="$ctrl.trafficInformations.traffic.data.usage.isThrottled"
+                data-ng-class="{ 'oui-badge_error': $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.overQuota }"
+            >
+                <span data-translate="server_consumption_throttled"></span>
+            </span>
+        </div>
+        <div class="oui-tile__item">
+            <dl class="oui-tile__definition">
+                <dt
+                    class="oui-tile__term"
+                    data-ng-bind="'server_consumption_current_period_label' | translate:{ t0: $ctrl.trafficInformations.traffic.data.usage.outputQuotaSize.text }"
+                ></dt>
+                <dd
+                    class="oui-tile__description"
+                    data-ng-if="$ctrl.trafficInformations.traffic.data.usage.outputQuotaUsed.value"
+                >
+                    <div class="oui-progress">
+                        <div
+                            class="oui-progress__bar oui-progress__bar_info"
+                            role="progressbar"
+                            aria-valuemin="0"
+                            aria-valuemax="{{ $ctrl.trafficInformations.traffic.data.usage.outputQuotaSize.value }}"
+                            aria-valuenow="{{ $ctrl.trafficInformations.traffic.data.usage.outputQuotaUsed.value }}"
+                            data-ng-style="{ 'width': $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.percent + '%' }"
+                            data-ng-class="{
+                                    'oui-progress__bar_warning': $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.nearQuota,
+                                    'oui-progress__bar_error': $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.overQuota
+                                 }"
+                        >
+                            <span
+                                class="oui-progress__label"
+                                data-ng-bind-template="{{ $ctrl.trafficInformations.traffic.data.usage.outputQuotaUsed.text }}"
+                            ></span>
+                        </div>
+                    </div>
+                    <span
+                        data-ng-bind="'server_consumption_current_period_remaining' | translate:{ t0: $ctrl.trafficInformations.traffic.data.usage.remainingOutputQuotaSize.text }"
+                    ></span>
+                </dd>
+            </dl>
+            <div
+                class="oui-tile__actions"
+                data-ng-if="$ctrl.canOrderTraffic() && $ctrl.canOrderMoreTraffic()"
```

Comment:
```suggestion
                data-ng-if="$ctrl.canOrderMoreTraffic()"
```
`$ctrl.canOrderMoreTraffic()` already checks `$ctrl.canOrderTraffic()` condition

Reply:
it's applied.